### DISCUSSION TITLE
Would it be possible to add a patch to disable location restriction in tiktok?

### DISCUSSION THREAD

**Discussion by P4R1H** [ReVanced/revanced-discussions#776] [Question]
tt uses the sim to determine location so that even vpns don't work to evade it, any way to disable it would be great

**Comment by d4rkk3y** [comment]
I am not sure but it may be possible. We need patch all the call to api that get the sim information. The problem is I don't know how to test if the patch work because I am not in restricted region.

**Reply by TheMerret** [reply]
I'm from restricted region, patch works well. Should I leave comment under PR?

**Reply by P4R1H** [reply]
> I'm from restricted region, patch works well. Should I leave comment under PR?

wow great yall really are wizards doing it this fast

**Reply by d4rkk3y** [reply]
> I'm from restricted region, patch works well. Should I leave comment under PR?

Thanks. I marked the PR ready for review.

**Comment by P4R1H** [comment]
in your team of devs there might be a couple people from the restricted
countries isn't it?

**Reply by d4rkk3y** [reply]
If you can build from source try this https://github.com/revanced/revanced-patches/pull/1088 and https://github.com/revanced/revanced-integrations/pull/219

**Comment by P4R1H** [comment]
nope, sorry i dont know how to

i'd love to help though so if there's any guide on how to i could try
